src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardRetryingRequestResolver.java

```diff
@@ -162,7 +169,19 @@ public void update()
             Integer currentAttempt = assignedRequests.get(t);
 
             this.setCurrent(t);
-            final IToken<?> resultingResolver = manager.reassignRequest(t, blackList);
+            final IToken<?> resultingResolver;
+
+            try
+            {
+                resultingResolver = manager.reassignRequest(t, blackList);
+            }
+            catch (Exception ex)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)